### ISSUE TITLE
Is it work still?

### ISSUE THREAD

**Issue by yadaniyil** [HeadLightSecurity/vkmitm#7]
I've tried to run script on not HTTPS connection and nothing was appear on terminal

**Comment by darker0n** [comment]
yes. it works now. But not works in ios app cause https **always** on. and vk also can change special bytes in `client<->server` protocol.

**Comment by yadaniyil** [comment]
What trouble could appear, if I've done all by the instruction and there was no messages on my terminal?

**Comment by darker0n** [comment]
@YaDaniil try to catch your vk messages. just send messages from local pc and start vkmitm.

**Comment by yadaniyil** [comment]
I've tried, but nothing appeared

**Comment by darker0n** [comment]
@YaDaniil Are you sure that https is disabled?

**Comment by yadaniyil** [comment]
Yea, I'm sure. All was done with the video instruction.

**Comment by darker0n** [comment]
@YaDaniil oh, i don't know. maybe vk has changed special bytes :confused:

**Comment by yadaniyil** [comment]
I'll try again, but if it's true, it's really bad(

**Comment by darker0n** [comment]
@YaDaniil take a moment. i'll check. if it's true I will push a patch.

**Comment by darker0n** [comment]
@YaDaniil It works for me.

**Comment by Merkif** [comment]
Работает ли данная уязвимость сейчас?
Is it work still?
